gnomad_qc/v3/analysis/vary_subpop_params.py

```diff
@@ -42,6 +44,15 @@ def main(args):  # noqa: D103
         # Initiate lists for storing results
         merged = []
 
+        if args.remove_outliers:
+            if not file_exists(subpop_outliers(pop).path):
+                raise DataException(
+                    f"The --remove-outliers option was used, but a Table of outlier samples does not exist for population {pop} at {subpop_outliers(pop).path}"
+                )
+                outliers_ht = subpop_outliers(pop).ht()
+            else:
+                outliers_ht = None
```

Comment:
Import `check_file_exists_raise_error` instead of `file_exists`:
```suggestion
        if args.remove_outliers:
            check_file_exists_raise_error(
                subpop_outliers(pop).path, 
                error_if_not_exists=True, 
                error_if_not_exists_msg=f"The --remove-outliers option was used, but a Table of outlier samples does not exist for population {pop} at {subpop_outliers(pop).path}"
            )
            outliers_ht = subpop_outliers(pop).ht()
        else:
            outliers_ht = None
```